app/assets/stylesheets/custom.css.scss

```diff
@@ -113,6 +117,7 @@ h1 {
   background-color: darkgrey;
 }
 #flash_message{
+  font-family: monospace;
```

Comment:
Properties should be ordered background-color, border-radius, color, font-family, margin-right, opacity, padding, text-align, text-color, text-style